src/mumble/ConnectDialog.cpp

```diff
@@ -89,10 +89,12 @@ ServerView::ServerView(QWidget *p) : QTreeWidget(p) {
 		siPublic = new ServerItem(tr("Public Internet"), ServerItem::PublicType);
 		siPublic->setChildIndicatorPolicy(QTreeWidgetItem::ShowIndicator);
 		addTopLevelItem(siPublic);
-		
-		
+
 		siPublic->setExpanded(false);
-	
+
+		// The continent code is empty when the server's IP address is not in the GeoIP database
+		qmContinentNames.insert(QLatin1String(), tr("Unknown"));
```

Comment:
Why isn't this sufficient to make everything work?

Reply:
Because that would still leave the country with the empty name inside the continent.